erc/ERCS/erc-7562.md

```diff
@@ -291,6 +291,9 @@ To help make sense of these params, note that a malicious paymaster can at most
         * The [RIP-7212](./rip-7212.md) sec256r1 precompile, on networks that accepted it.
 * **[OP-070]** Transient Storage slots defined in [EIP-1153](./eip-1153) and accessed using `TLOAD` (`0x5c`) and `TSTORE` (`0x5d`) opcodes
   are treated exactly like persistent storage (SLOAD/SSTORE).
+* **[OP-080]** Allow access to `BALANCE` (`0x31`) and `SELFBALANCE` (`0x47`) from a staked entity. \
+These opcodes remain blocked for unstaked entites.
```

Comment:
I guess we need to remove them from op-010, otherwise there is a conflict ..

Reply:
Makes sense, going to fix this for OP-012 as well.